cvat/schema.yml

```diff
@@ -11039,6 +11058,14 @@ components:
           allOf:
           - $ref: '#/components/schemas/StorageRequest'
           nullable: true
+        consensus_replicas:
```

Comment:
Perhaps this could be called something like `consensus_replicas_per_job`, to distinguish it from the `consensus_replicas` field in the job?

Reply:
It was called this way previously, but it looks very clumsy to me. Adding "per job" doesn't change it's meaning, because only jobs can have replicas within a task, so I decided to keep it shorter.

Reply:
Alright, let's keep it.